### PR TITLE
Monorepo: Enable plain go build

### PR DESCRIPTION
**Pull request by mandarjog**
**What this PR does / why we need it**:
Add top level bin/init.sh to setup go build environment.
presubmit ensures go buildability.  

This code can be refactored later.

```release-note
NONE
```

### CODE REVIEW

File: bin/init.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+bazel build //...
+
+source "${ROOT}/bin/use_bazel_go.sh"
+
+# Clean up vendor dir
+rm -rf ${ROOT}/vendor
+mkdir -p ${ROOT}/vendor
+
+# Vendorize bazel dependencies
+${ROOT}/mixer/bin/bazel_to_go.py ${ROOT}
```
Comment (rshriram) [inline bin/init.sh]: you also need to run the bin/init.sh equivalent in pilot.

Comment (mandarjog) [reply bin/init.sh]: Mixer bazel to py is. Super set of others.
Just with this, pilot mixer security and broker all ‘go build’ fine.

Comment (rshriram) [reply bin/init.sh]: but that path fix script below is only fixing mixer paths.. What about pilot paths?

**Comment by sebastienvas** [comment]
Can we use this opportunity to have only one bazel_to_go for all ?  the one from auth, pilot and istio are pretty simple, only mixer is complicated.

**Comment by mandarjog** [comment]
Only one is used now the one from mixer. 
I will remove the other ones and move things around in another PR. This is least amount of change to get things going.

**Pull request merged**